FreeRTOS/FreeRTOS
# Update win32 demo to exercise configRUN_TIME_COUNTER_TYPE

Description
-----------
PR https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/350 introduced configRUN_TIME_COUNTER_TYPE and ulTaskGetIdleRunTimePercent().  This PR updates the Win32 demo to exercise both additions.

Must be merged after https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/350


Test Steps
-----------
Run the full Win32 demo.
Used new MPU function in STM32 MPU port.

Related Issue
-----------
https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/350


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
